Makefile

```diff
@@ -46,9 +46,9 @@ setup:
 	fi \
 
 	if [[ ! -z "$$EVENT_NAME" ]] && [[ "$$EVENT_NAME" == PR ]]; then \
-		pip install bc-python-hcl2==0.3.15 cfn-lint==0.47.2 pre-commit flake8 flake8-print checkov==1.0.580;\
+		pip install bc-python-hcl2==0.3.15 cfn-lint==0.47.2 networkx==2.5 pre-commit flake8 flake8-print checkov==1.0.580;\
```

Comment:
There is a newer version available - https://pypi.org/project/networkx/2.6.2/, any particular reason to use version2.5?

Reply:
To @BVyshnavi's point, was there testing done on versions above 2.5? A quick look at GitHub indicates that the changes from 2.5 to 2.6 are significant.

Reply:
@BVyshnavi and @sirhc ,
2.6 is breaking as mentioned in PR description with error details. Yes, tested with 2.5 few repositories and builds were successful, log links were given as well.

Reply:
Latest version of networkx is what broke the cfn-linter and this PR is to fix that pin the version of last working version which is 2.5.